$MirOS: ports/graphics/imlib2/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:46:54 tg Exp $
--- configure.in.orig	Fri Jun 10 02:24:37 2005
+++ configure.in	Wed Dec 28 18:28:11 2005
@@ -269,6 +269,15 @@ fi
 AM_CONDITIONAL(BUILD_TIFF_LOADER, test "$tiff_ok" = yes)
 AC_SUBST(TIFFLIBS)
 
+AC_ARG_ENABLE([gif],
+[  --enable-gif                              enable libungif/libgif checks [yes]],
+[gif_check="$enableval"],
+[gif_check=yes])
+
+if test "$gif_check" = no; then
+AC_MSG_WARN(*** Native GIF support will not be built (manually disabled) ***)
+gif_ok=no
+else
 AC_CHECK_LIB(ungif, DGifOpenFileName,
   gif_libs="-lungif"
   gif_ok=yes,
@@ -312,6 +321,7 @@ if test "$gif_ok" = no; then
       AC_MSG_WARN(*** Native GIF support will not be built (GIF header file not found) ***)
     fi
   fi
+fi
 fi
 AM_CONDITIONAL(BUILD_GIF_LOADER, test "$gif_ok" = yes)
 AC_SUBST(GIFLIBS)
